web/wp-content/themes/dctx-v2/inc/scaffolding.php

```diff
@@ -0,0 +1,257 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-n0iilc.php, but found phpcs-scan-N0IiLc.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).